diff --git a/CMake/FindMHD.cmake b/CMake/FindMHD.cmake
index 822714a..33fbe78 100644
--- a/CMake/FindMHD.cmake
+++ b/CMake/FindMHD.cmake
@@ -34,5 +34,5 @@ if (MSVC)
 endif()
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(mhd DEFAULT_MSG MHD_INCLUDE_DIR MHD_LIBRARY)
+find_package_handle_standard_args(MHD DEFAULT_MSG MHD_INCLUDE_DIR MHD_LIBRARY)
 mark_as_advanced(MHD_INCLUDE_DIR MHD_LIBRARY)
